crates/astria-sequencer/src/sequencer.rs

```diff
@@ -28,9 +28,20 @@ impl Sequencer {
     pub async fn run_until_stopped(config: Config) -> Result<()> {
         let genesis_state =
             GenesisState::from_path(config.genesis_file).context("failed reading genesis state")?;
-        let storage = penumbra_storage::TempStorage::new()
+        if config.db_filepath.try_exists()? {
+            info!(
+                path = config.db_filepath.display().to_string(),
```

Comment:
No need to allocate:
```suggestion
                path = %config.db_filepath.display(),
```